### PR TITLE
update zenodo file and README

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- Update contributors for 0.7.0
`git log 0a885926683b960badbd5265bcd25e6c4e90d384..HEAD | grep Author | sort | uniq | sed "s/ \+/ /g; s/^ //" | cut -d" " -f2-3 | uniq`
- add missing names to README.txt thanks section

### CODE REVIEW

**Comment by BrianMarre** [comment]
good to know that I am a developer, now ;)

File: .zenodo.json [hunk 1]
```diff
@@ -47,43 +48,40 @@
...
-      "affiliation":"HZDR"
+      "name":"Marre, Brian Edward",
+      "affiliation":"HZDR, TU Dresden"
     },
     {
       "name":"Ordyna, Paweł",
       "affiliation":"HZDR, TU Dresden"
     },
+    {
+      "affiliation":"HZDR, TU Dresden",
```
Comment (PrometheusPi) [inline .zenodo.json]:
```suggestion
      "affiliation":"HZDR",
```

File: .zenodo.json [hunk 2]
```diff
@@ -195,11 +193,51 @@
...
+      "type":"Other"
+    },
+    {
+      "affiliation":"LogMeIn, Inc.",
+      "name":"Worpitz, Benjamin",
+      "type":"Other"
+    },
+    {
+      "name":"Wrobel, Nico",
+      "affiliation":"HZDR",
```
Comment (PrometheusPi) [inline .zenodo.json]:
```suggestion
      "affiliation":"HZDR, TU Dresden",
```

File: .zenodo.json [hunk 3]
```diff
@@ -47,43 +48,40 @@
...
     },
     {
-      "name":"Thévenet, Maxence",
-      "affiliation":"Desy"
-    },
-    {
-      "affiliation":"HZDR, TU Dresden",
-      "name":"Pausch, Richard",
-      "orcid":"0000-0001-7990-9564"
+      "affiliation":"HZDR",
```
Comment (PrometheusPi) [inline .zenodo.json]:
```suggestion
      "affiliation":"HZDR, TU Dresden",
```

**Review by PrometheusPi — CHANGES_REQUESTED** [review]
some minor changes are required
and the TU is sometimes written as `TU Dresden` and sometimes as `TU-Dresden`. 
Perhaps this should be consistent.

**Comment by psychocoderHPC** [comment]
@PrometheusPi I fixed the affiliations

File: .zenodo.json [hunk 4]
```diff
@@ -6,18 +6,23 @@
...
+      "affiliation":"HZDR, TU-Dresden",
+      "name":"Carstens, Finn-Ole"
     },
     {
       "affiliation":"HZDR",
       "name":"Debus, Alexander",
       "orcid":"0000-0002-3844-3697"
     },
+    {
+      "affiliation":"HZDR",
```
Comment (PrometheusPi) [inline .zenodo.json]:
```suggestion
      "affiliation":"HZDR, TZ-Dresden",
```

Sorry, I did not see that before

File: .zenodo.json [hunk 5]
```diff
@@ -161,23 +159,23 @@
       "type":"Other"
     },
     {
-      "affiliation":"ELI-NP",
-      "name":"Ong, Jian Fuh",
+      "name":"Meyer, Felix",
+      "affiliation":"HZDR",
```
Comment (PrometheusPi) [inline .zenodo.json]:
```suggestion
      "affiliation":"HZDR, TU Dresden",
```

File: .zenodo.json [hunk 6]
```diff
@@ -161,23 +159,23 @@
...
-      "affiliation":"HZDR, TU Dresden",
-      "name":"Rudat, Sophie",
+      "affiliation":"ELI-NP",
+      "name":"Ong, Jian Fuh",
       "type":"Other"
     },
     {
-      "affiliation":"TU-Dresden",
-      "name":"Winkler, Frank",
+      "affiliation":"HZDR",
```
Comment (PrometheusPi) [inline .zenodo.json]:
```suggestion
      "affiliation":"HZDR, TU Dresden",
```

**Review by PrometheusPi — CHANGES_REQUESTED** [review]
Sorry @psychocoderHPC I found some more missing TU DD affiliations.

**Review by PrometheusPi — APPROVED** [review]

**Pull request merged**